deps/zlib/zconf.h.cmakein

```diff
@@ -0,0 +1,549 @@
+/* zconf.h -- configuration of the zlib compression library
```

Comment:
We have a `deps/zlib/zconf.h`. If upstream Chromium forked zlib now generates `zconf.h` from a template, are we now out of step? Do we need to update the maintaining guide?

Reply:
Was `deps/zlib/zconf.h--` from upstream? Were there no `zconf.h` changes we should be picking up?

Reply:
Yes to both questions.

Reply:
On second look, `deps/zlib/zconf.h--` seems the result of running this command

```
sed -i -- 's_^#include "chromeconf.h"_//#include "chromeconf.h"_' deps/zlib/zconf.h
```

on macOS.

Reply:
Will fix.